README.md

```diff
@@ -0,0 +1,310 @@
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+
+### Prerequisites
+
+In order to run this project you need:
+```sh
+ Visual Studio Code
+```
+ 
+ ```sh
+ Git bash
+```
+ 
+-->GitHub<br>
+-->Read the materials that was provided by microverse
+
+<!--
+Example command:
+
+```sh
+ gem install rails
+```
+ -->
+
+### Setup
+
+-Create a new GitHub repo.<br>
+-Create a new branch according to the GitHub flow rules.<br>
+-Add a .gitignore file.<br>
+-Add node_modules to gitignore file.<br>
+-Set up a linter for HTML and CSS and make sure that any files that are not relevant are ignored by git.<br>
+-Add descriptive README file to your project - please use this template but remember to customize it to your project.<br>
+-Add 3 files (each one in a separate commit):<br>
+        -A .html file that includes a header with the text "Hello Microverse!". Use h1 tag for that.<br>
+        -A .css file that includes one class with styles for your header (make it in your favorite color).<br>
+        
+-Make sure that the last file is actually ignored by git and is not present in your GitHub repository.<br>
+-Open a pull request<br>
+-Set up a new repository and prepare it for development using best practices (e.g. linters).
+-Build the first 2 sections of the mobile website using the template you chose in the previous step.
+-In order to lay out the elements on the page you should use flexbox in all 2 sections.
+-Only use HTML & CSS (no JavaScript needed)
+-Your website should look exactly like the design (e.g. font, colors, images, text, margins) using the templates in Figma.
+-Implement the button interactions (i.e. hover, etc.).
+-he website should be responsive, following these breakpoints:
+-Minimum mobile screen size should be 375px
+-Minimum desktop screen size should be 768px
+-Use responsive values in your CSS rules, like percentages (i.e. width: 80%), instead of pixels
+
+
+<!--
+Example commands:
+
+```sh
+  cd my-folder
+  git clone git@github.com:myaccount/my-project.git
+```
+--->
+
+### Install
+
+Install this project with: GitHub
+
+<!--
+Example command:
+
+```sh
+  cd my-project
+  gem install
+```
+--->
+
+### Usage
+
+To run the project, execute the following command:
+```sh
+  PS C:\Users\helin\OneDrive\Documents\GitHub\Portfolio setup and mobile>
+```
+
+<!--
+Example command:
+
+```sh
+  rails server
+```
+--->
+
+### Run tests
+
+To run tests, run the following command:
+```sh
+  npx hint .
+```
+```sh
+  npx stylelint "**/*.{css,scss}"
+```
+
+<!--
+Example command:
+
+```sh
+  bin/rails test test/models/article_test.rb
+```
+--->
+
+### Deployment
+
+You can deploy this project using:
+
+<!--
+Example:
+
+```sh
+
+```
+ -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Author1**
+
+- GitHub: [@helina42](https://github.com/helina42)
+
+
```

Comment:
- [ ] Please write your name into the 'author1' title, also, add your Twitter and LinkedIn accounts as well so that they can reach you.